test/wasmBenchmarker/benchmark.py

```diff
@@ -30,31 +31,31 @@
 DIFF_TRESHOLD = 1e-7
 
 expectedValues = {
-    "change": 4,
-    "factorial": 30,
-    "fannkuch": 120,
-    "fibonacci": 1,
+    "change": 3,
+    "factorial": 899999994000000000,
+    "fannkuch": 360,
+    "fibonacci": 63245986,
     "gregory": 3.14159264,
     "hanoi": 0,
     "heapsort": 0,
     "huffman": 0,
-    "kNucleotide": 1,
-    "mandelbrotFloat": 775007,
-    "mandelbrotDouble": 775007,
+    "kNucleotide": 0, # 1
```

Comment:
is this a bool? Could we have a better check for successful run?

Reply:
Yes, it is a bool, but the check function is in the test. So, the test runs the algorithm, and after that it checks the values. 
https://github.com/Samsung/walrus/blob/375d6886e083a2c67eb451ffee10b98dfb8f42e5/test/wasmBenchmarker/ctests/kNucleotide.c#L190